### ISSUE TITLE
Sending incomplete

### ISSUE THREAD

**Issue by Bitc0inare** [breadwallet/breadwallet#8]
Hello I have had the Bread Wallet for a month now and I do love the simplicity and beautiful well designed platform. However after having success the 1st sending/receiving of payments, it now seems to not want to send/receive.  I am having the same problem as this person had stated "The app was working great until a few days ago. Now the majority of my payments never complete. I have only made 1 successful payment out of the last 10. Maybe it is a bug from the IOS 8.0.2 update. I have a a concerning amount of BTC stuck in this wallet so hopefully it gets fixed soon. When the bug is fixed I will probably bump my rating up to 5 stars.

*I am going to update my phone with the new 8.1.2 recent update and hopefully this will fix the bug.  If not, please do give me support.  I need my money out of the wallet ASAP so that i can send it off to a trading platform.

Thank-You!

**Comment by voisine** [comment]
new update release should make this happen far less, and will alert the user to rescan if detected.

**Comment by Bitc0inare** [comment]
Hey Aaron, long time no hear! Last time we communicated I was having an
issue with 'breadwallet' and here I am yet again with another issue I've
been experiencing for the last few months or so. The issue I'm having now
is that whenever I click on 'pay address from clipboard', it states
"clipboard doesn't contain a valid bitcoin address" - Any particular reason
for this happening and how can I avoid this issue? I rarely use this
feature but it comes in handy@times.  I have an email address I would like
for you to contact me back with and it is - bitcoinaire35@icloud.com, if
you will!
I look forward for your response, thank-you!

bitcoinaire35@icloud.com > And yes, I did a rescan of the blockchain but
problem still persist. May try again and will let you know the result....
